### PR TITLE
watch tasks bug

### LINKED ISSUE #17: Path bug in watch task
```json
"watch:js": "chokidar 'source/js/*.js' -c 'npm run build:js'",
"watch:pug": "chokidar 'source/pug/*/**.pug' -c 'npm run build:pug'",
"watch:sass": "chokidar 'source/sass/*.scss' -c 'npm run build:sass'"
```
We should be watching changes in all `.js` file under the `source/js` directory and `.scss` files under the `source/scss` instead. (not just files in the first level dir under it)

### PR DESCRIPTION
**Pull request by mmmavis**
Fixes #17

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -19,9 +19,9 @@
     "test:scss": "stylelint 'source/sass/*.scss' --syntax scss",
     "test": "run-s test:**",
     "watch:images": "chokidar 'source/images/*' -c 'npm run build:images'",
-    "watch:js": "chokidar 'source/js/*.js' -c 'npm run build:js'",
-    "watch:pug": "chokidar 'source/pug/*/**.pug' -c 'npm run build:pug'",
-    "watch:sass": "chokidar 'source/sass/*.scss' -c 'npm run build:sass'"
+    "watch:js": "chokidar 'source/js/*/*.js' -c 'npm run build:js'",
+    "watch:pug": "chokidar 'source/pug/*/*.pug' -c 'npm run build:pug'",
+    "watch:sass": "chokidar 'source/sass/*/*.scss' -c 'npm run build:sass'"
```
Comment (ScottDowne) [inline package.json]: Does this work on files like main.js, main.scss that are only one directory deep?

Comment (mmmavis) [reply package.json]: oh shoot. should be `**`. fixing

**Comment by ScottDowne** [comment]
Seems to work now, but travis looks to be failing now.

**Comment by mmmavis** [comment]
yea it's all the linting errors that we would've caught if the `test` task had worked as expected 😉 

I will fix those linting errors now.

**Comment by mmmavis** [comment]
okay now I get ✅ from Travis

**Comment by ScottDowne** [comment]
Works for me.

**Comment by mmmavis** [comment]
k merging

**Pull request merged**